### ISSUE TITLE
New model for QuickLinks...

### ISSUE THREAD

**Issue by simoncorry** [visiblemiles/frontend#28]
Currently `QuickLinks` lives in the header with a whole bunch of conditional logic. Let's move it out of header and into individual page layouts. You'll see an example how this works in `organisms/UserProfile`.

- I've created a more robust version of `molecules/QuickLinks` called `molecules/PageLinks` (you can change the name). The idea being that `PageLinks` will replace `QuickLinks` when you've ported the necessary logic over.

- `PageLinks` are sticky and stick to the page below the header when you scroll.

- Every page in this new paradigm would have a `PageLinks` and depending on the type of page it would either show the quick links values (i.e. Mac, pc, PS4, etc) or the page navigation values (i.e. Profile, Badges, Bookmarks, etc)

Here's how it would look:
- Home page = Quick Links Values
- Article page = Quick Links Values
- Tag/category/search results pages = Quick Links Values
- Profiles user/games/companies = Page Navigation Values
- Company = Page Navigation Values
- Settings = Page Navigation Values

In summary the new `molecules/PageLinks` is cool and it solves a lot of weird edge cases.

**Comment by EricWVGG** [comment]
note to self: "quick links values" means "default values", would be nice to content manage this

**Comment by simoncorry** [comment]
Would be nice to have an ifProp showing whether the bar is displaying default values so I can hide/show the label at the far left.

<img width="195" alt="Screen Shot 2019-07-03 at 11 23 01 AM" src="https://user-images.githubusercontent.com/691178/60604106-f4370080-9d84-11e9-841f-f33efa0ca3c1.png">
